patterns/definitions.owl

```diff
@@ -105186,6 +105187,13 @@ AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#has
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/OBA_2050418> "voice quality trait"^^xsd:string)
 EquivalentClasses(<http://purl.obolibrary.org/obo/OBA_2050418> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0001519> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000052> <http://purl.obolibrary.org/obo/UBERON_0034681>)))
 
+# Class: <http://purl.obolibrary.org/obo/OBA_2050419> (level of endocrine disruptor in urine)
```

Comment:
@matentzn , Is this addition necessary in this PR?

Reply:
Not necessary, it just happened when I ran the test. Feel free to remove.